distribution/docs/src/main/resources/content/_metadataAttributes/isr-attributes-table.adoc

```diff
@@ -89,8 +89,8 @@
 
 |isr.cloud-cover
 |Identifies the percentage of the image that is obscured by clouds.
-|Integer
-|0-100
+|Double
+|0.0-100.0
```

Comment:
Does the ISR attribute need to be updated too?
https://github.com/codice/alliance/blob/d559287819156c69ff451b45b67e78f8ac7a94f1/catalog/core/catalog-core-api-impl/src/main/java/org/codice/alliance/catalog/core/api/impl/types/IsrAttributes.java#L32-L39